### PR TITLE
MGMT-10420: Extra spaces in installation page

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-10420

I have slightly decreased the spacing in and around the cluster progress item and also between the alerts. It's not exactly by half as the design says in the following image but this is the best that can be achieved without overwriting Patternfly variables.

image

### Before and after images:

**Before:**
image

**After:**
image

---

**Before:**
image

**After:**
image

---

**Before:**
image

**After:**
image

### CODE REVIEW

**Comment by nirfarkas** [comment]
Looks great! Thanks!

File: src/common/components/clusterDetail/ProgressBarTexts.tsx [hunk 1]
```diff
@@ -25,6 +24,7 @@ export const ProgressBarTexts: React.FC<HostProgressProps> = ({ hosts, hostRole
           filteredHosts.length,
           hostRole === 'master' ? 'control plane node' : 'worker',
         )}`;
+
   const getHostName = (hostRole: HostRole): string => {
```
Comment (jkilzi) [inline src/common/components/clusterDetail/ProgressBarTexts.tsx:28]: Should be declared outside the component

File: src/common/components/clusterDetail/ProgressBarTexts.tsx [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 type HostProgressProps = {
   hosts: Host[];
   hostRole: HostRole;
 };
 
-export const ProgressBarTexts: React.FC<HostProgressProps> = ({ hosts, hostRole }) => {
+export const ProgressBarTexts = ({ hosts, hostRole }: HostProgressProps) => {
   const filteredHosts = hosts.filter((host) => host.role && hostRole === host.role);
   const failedHostsCount = filteredHosts.filter((host) => host.status === 'error').length;
   const hostCountText = (hostRole: HostRole) =>
```
Comment (jkilzi) [inline src/common/components/clusterDetail/ProgressBarTexts.tsx:17]: Cab be declared outside the component, just pass in the `filteredHosts` and `failedHostsCount` as additional arguments.

**Review by jkilzi — CHANGES_REQUESTED** [review]

File: src/common/components/clusterDetail/ProgressBarTexts.tsx (hunk 1 above)
Comment (jgyselov) [reply src/common/components/clusterDetail/ProgressBarTexts.tsx]: It doesn't even need to be a function :grin:

**Review by jkilzi — DISMISSED** [review]

File: src/common/components/clusterDetail/ProgressBarTexts.tsx [hunk 3]
```diff
@@ -12,45 +12,39 @@ type HostProgressProps = {
 };
 
 export const ProgressBarTexts = ({ hosts, hostRole }: HostProgressProps) => {
-  const filteredHosts = hosts.filter((host) => host.role && hostRole === host.role);
-  const failedHostsCount = filteredHosts.filter((host) => host.status === 'error').length;
-  const hostCountText = (hostRole: HostRole) =>
+  const hostRoleText = pluralize(hosts.length, hostRole === 'master' ? 'Control Plane' : 'Worker');
+  const failedHostsCount = hosts.filter((host) => host.status === 'error').length;
+
+  const hostCountText =
```
Comment (celdrake) [inline src/common/components/clusterDetail/ProgressBarTexts.tsx:18]: I can only suggest a small change, to remove some duplicate code.

```
  const hostCountText = `${hosts.length} ${pluralize(
    hosts.length,
    hostRole === 'master' ? 'control plane node' : 'worker',
  )}`
```

then 
```
  if (hosts.some((host) => ['cancelled', 'error'].includes(host.status))) {
    const failedHostsCount = hosts.filter((host) => host.status === 'error').length;
    return (
      <ClusterProgressItem icon={<ExclamationCircleIcon color={dangerColor.value} />}>
        <>
          {hostRoleText}
          <br />
          <small>{failedHostsCount}/{hostCountText} failed</small>
        </>
      </ClusterProgressItem>
    );
  }
```

**Review by celdrake — COMMENTED** [review]
Good job with the refactor and extracting the new component!
Check my suggestion but it's pretty minor so it can be skiped :)

File: src/common/components/clusterDetail/ProgressBarTexts.tsx (hunk 3 above)
Comment (jgyselov) [reply src/common/components/clusterDetail/ProgressBarTexts.tsx:18]: Changed, thanks for the suggestion!

**Pull request merged**